torch/_tensor_str.py

```diff
@@ -38,7 +38,8 @@ def set_printoptions(
         profile: Sane defaults for pretty printing. Can override with any of
             the above options. (any one of `default`, `short`, `full`)
         sci_mode: Enable (True) or disable (False) scientific notation. If
-            None (default) is specified, the value is defined by `_Formatter`
+            None (default=False) is specified, the value is defined by 
```

Comment:
`None` is the default and means the framework makes a decision on whether or not to print with scientific notation. See https://github.com/pytorch/pytorch/blob/caaea084e9e63dd3f22edcab0fda6822f7634820/torch/_tensor_str.py#L117-L142 . We should keep this text as `None (default)` but make it more clear that the framework chooses when it happens.